### DISCUSSION TITLE
SelectTree是否也应该增加IsReset参数

### DISCUSSION THREAD

**Discussion by tiansfather** [dotnetcore/BootstrapBlazor#4862] [Q&A]
TreeView组件有个IsReset参数，默认为false,也就是必须设为true才能让Items重新生成Rows
SelectTree里也用到了TreeView，但是里面并未设置这个参数，也就是设置过一次Items之后想更改就不行了
对于SelectTree一般都是用于表单，是否需要默认设为true，或者对外暴露这个参数
![image](https://github.com/user-attachments/assets/50ae19ab-fa7b-464a-9680-7287a8ed487b)

**Comment by ArgoZhang** [comment]
@tiansfather 感谢，帮忙加上来个 `PR`

**Comment by tiansfather** [comment]
好的
